src/prebid.js

```diff
@@ -105,6 +105,19 @@ function setRenderSize(doc, width, height) {
   if (doc.defaultView && doc.defaultView.frameElement) {
     doc.defaultView.frameElement.width = width;
     doc.defaultView.frameElement.height = height;
+    // in case of nested iframes, resize those as well
```

Comment:
I guess there's no chance of resizing an iframe unrelated to the ad because it would be x-domain?

Reply:
it's possible I suppose if someone is using nested iframes for the website structure. I don't think that's very common with modern HTML though?